app/views/provider/admin/account/users/index.html.slim

```diff
@@ -38,12 +38,12 @@ table class="data" id="users"
             = user.sso_authorizations.exists? ? 'yes' : 'not yet'
         td = user.role
 
-        - if can?(:manage, :groups)
+        - if can?(:manage, :permissions)
```

Comment:
I think this is completely useless and we could probably remove it. I don't see a case where one can have access to this page and NOT be able to manage member permissions.

I kept the conditional for the time being only because it was there, except that this has nothing to do with "groups", which is a provider switch.